testing/python/issue/test_tilelang_issue_2527.py

```
@@ -0,0 +1,31 @@
+src = m.get_kernel_source()
+print("USES_TMA_STORE:", "tma_store" in src)  # -> True
+for line in src.splitlines():
+    if "tl::tma_store(" in line:
+        print("  ", line.strip())  # -> tl::tma_store(B_desc, (&(As[0])), 1, 0);
+
+B = torch.zeros((M, N), dtype=torch.int32, device="cuda")
+try:
+    m(B)
+    torch.cuda.synchronize()
+    print("default OK?", bool((B.cpu()[m0 : m0 + mm, n0 : n0 + nn] == 7).all()))  # -> region == 7
+except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)